test/VectorInstructions/extract_element.c

```diff
@@ -1,7 +1,7 @@
 // REQUIRES: geq-llvm-3.4
 // RUN: %llvmgcc %s -emit-llvm -O0 -g -c -o %t1.bc
 // RUN: rm -rf %t.klee-out
-// RUN: %klee --output-dir=%t.klee-out --exit-on-error %t1.bc
+// RUN: %klee --output-dir=%t.klee-out --optimize=false --exit-on-error %t1.bc
```

Comment:
With `--optimize` all the vector instructions get constant folded away, so we need to use `--optimize=false` to stop that from happening. I probably ought to add a comment about this.